src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -170,6 +170,10 @@ public void validateMandatoryPropertyErrorFor(String property) {
     CdfPluginPropertiesActions.verifyRequiredPropertyHasNoValueErrorMessage(property);
     CdfPluginPropertiesActions.verifyPluginPropertyInlineErrorMessageColor(property);
   }
+  @Then("Press ESC key to close the joiner fields dropdown")
```

Comment:
Make it generic as the same action is applied for all the Plugin properties page

Reply:
Made the statement generic